File: tests/conformance/fixtures/go/17-outputs.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("build").Run("make build").
		Output("binary", "/out/app").
		Output("checksum", "/out/sha256")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/17-outputs.exs
use Sykli

pipeline do
  task "build" do
    run "make build"
    output "binary", "/out/app"
    output "checksum", "/out/sha256"
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
